### PR TITLE
[Documentation:Developer] Reorganize troubleshooting

### PR DESCRIPTION
**Pull request by RitaLei123**
Reorganized the troubleshooting pages under developer. Organized troubleshooting into 3 files (Installation troubleshooting, Re-installation troubleshooting, and System Clock Testing & Troubleshooting). Now developers can go to the new troubleshooting folder to find instructions on how to fix their problem.

image

### CODE REVIEW

**Review by Nicc041 — APPROVED** [review]
Everything works and the code looks good.

**Review by benjaminlee755 — COMMENTED** [review]
The code seems fine and works.

File: _docs/developer/development_instructions/index.md [hunk 1]
```diff
@@ -204,133 +205,6 @@ autograding configuration, you'll probably need to:
 
 ---
 
-## System Clock Testing & Troubleshooting
```
Comment (bmcutler) [inline _docs/developer/development_instructions/index.md:207]: i agree with moving the system clock troubleshooting to another file

File: _docs/developer/development_instructions/index.md [hunk 2]
```diff
@@ -204,133 +205,6 @@ autograding configuration, you'll probably need to:
...
-  To sync back with the current time:
-
-  ```
-  sudo systemctl enable ntp
-  timedatectl set-ntp 1
-  ```
-
----
-
-## System Re-Configuration
```
Comment (bmcutler) [inline _docs/developer/development_instructions/index.md:243]: but the instructions for teting changes to configure_submitty.py & etc should stay here

File: _docs/developer/development_instructions/index.md [hunk 3]
```diff
@@ -204,133 +205,6 @@ autograding configuration, you'll probably need to:
...
-* To update existing databases:
-
-  ```
-  sudo python3 /usr/local/submitty/GIT_CHECKOUT/Submitty/.setup/update_database.py
-  ```
-
-
----
-
-## Re-Creating All Sample Course Data
```
Comment (bmcutler) [inline _docs/developer/development_instructions/index.md:266]: and the instructions for rebuilding sample course data should stay here

File: _docs/developer/development_instructions/index.md [hunk 4]
```diff
@@ -204,133 +205,6 @@ autograding configuration, you'll probably need to:
...
-
-
-  NOTE: This command will also have to be run twice a year on July 1st and January 1st when the test semester will change from fall to spring or vice versa.
-
-
-  See also: [Sample Course Data](/developer/development_instructions/sample_data)
-
----
-
-## Complete System Re-Installation
```
Comment (bmcutler) [inline _docs/developer/development_instructions/index.md:299]: and system re-installation instructions should stay here

File: _docs/developer/development_instructions/sample_data.md [hunk 5]
```diff
@@ -72,5 +72,4 @@ You may also need to edit test cases in Cypress, Selenium, etc.
 
 ---
 
-See also: [Re-Creating All Sample Course Data](/developer/development_instructions/index#re-creating-all-sample-course-data)
-
+See also: [Re-Creating All Sample Course Data](/developer/troubleshooting/reinstallation_troubleshooting)
```
Comment (bmcutler) [inline _docs/developer/development_instructions/sample_data.md:75]: so we will revert this change

File: _docs/developer/troubleshooting/reinstallation_troubleshooting.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
+---
```
Comment (bmcutler) [inline _docs/developer/troubleshooting/reinstallation_troubleshooting.md:1]: this shouldn't be a separate page, its not troubleshooting, they are development instructions (less commonly used, but still development instructions)

File: navtreedata.js [hunk 7]
```diff
@@ -234,6 +233,11 @@ var NAVTREE =
                     [ "Frontend JavaScript", "/developer/developing_the_php_site/javascript", null]
                 ] ],
             ] ],
+            [ "Troubleshooting & Other Information", "/developer/troubleshooting/installation_troubleshooting", [
+                [ "Installation", "/developer/troubleshooting/installation_troubleshooting", null ],
+                [ "Re-Installation", "/developer/troubleshooting/reinstallation_troubleshooting", null ],
```
Comment (bmcutler) [inline navtreedata.js:238]: I think this goes away

**Review by bmcutler — CHANGES_REQUESTED** [review]
suggested undoing some of the shuffling

**Review by bmcutler — APPROVED** [review]

**Pull request merged**